BUILD.gn

```diff
@@ -29,3 +29,11 @@ copy("copy_old_sign_update") {
     "$root_out_dir/old_dsa_scripts/{{source_file_part}}",
   ]
 }
+
+action("tar_sparkle_binaries") {
+  script="tar_sparkle_binaries.py"
```

Comment:
It would be better to pass source files list as an args and source files list could be variable that can be used with `build_sparkle_framework` target?

Reply:
Hi Simon, why would that be better? My reasoning is that this repo knows which binaries it can offer; and that I would imagine that passing the list in from outside would complicate the code.

Reply:
My intention was removing duplication.
output list of `build_sparkle_framework` and input list of `tar_sparkle_binaries.py` will always same.
When we update one list, the other should be updated also.

Of course, if it's not update frequently, I think current PR seems fine.
This is trivial, So, I think it's up to you. :)

Reply:
Or adding a comment about this would be sufficient.

Reply:
@simonhong I misunderstood your original comment and tried to address it in f69ba7b7fae7dda475ba6cda12fba8d72270478d. What do you think?